pkg/bpf/bpf.go

```diff
@@ -89,11 +89,11 @@ func Start() error {
 		return err
 	}
 
-	if config.EnableKmesh {
+	if config.AdsEnabled() {
 		if err = StartKmesh(); err != nil {
 			return err
 		}
-	} else if config.EnableKmeshWorkload {
+	} else if config.WdsEnabled() {
 		if err = StartKmeshWorkload(); err != nil {
 			return err
 		}
```

Comment:
It would be better to return error, if config is neither ads or workload. It would be more robust in `start()` stage